akka-typed/src/main/scala/akka/typed/scaladsl/Actor.scala

```diff
@@ -273,11 +273,11 @@ object Actor {
    * change is desired, use `Actor.same()`.
    */
   final case class SignalOrMessage[T](
-    signal: (ActorContext[T], Signal) ⇒ Behavior[T],
-    mesg:   (ActorContext[T], T) ⇒ Behavior[T]) extends Behavior[T] {
+    signal:  (ActorContext[T], Signal) ⇒ Behavior[T],
+    message: (ActorContext[T], T) ⇒ Behavior[T]) extends Behavior[T] {
     override def management(ctx: AC[T], msg: Signal): Behavior[T] = signal(ctx, msg)
-    override def message(ctx: AC[T], msg: T): Behavior[T] = mesg(ctx, msg)
-    override def toString = s"SignalOrMessage(${LineNumbers(signal)},${LineNumbers(mesg)})"
+    override def message(ctx: AC[T], msg: T): Behavior[T] = message(ctx, msg)
```

Comment:
This is what I also started out from, to discover quickly that it is nothing but an endless loop ;-)